blocks/melody/_shared/mixins.less

```diff
@@ -114,11 +114,37 @@
     }
 }
 
-
+// Calculating contrasts from here: http://24ways.org/2010/calculating-color-contrast/
+.colour-guard(@colour, @dark: #000000, @light: #ffffff) when ((((red(@colour)*299)+(green(@colour)*587)+(blue(@colour)*114))/1000) > 204) {
```

Comment:
These probably shouldn't be in a non-prefixed context; they're only useful for text for so could go somewhere in prose utils.